src/_common/M.js

```diff
@@ -15,11 +15,13 @@ export default class M extends Component {
 		const { m = '', values } = this.props;
 
 		return (
-			<FormattedMessage
-				id={m}
-				defaultMessage={m}
-				values={values}
-			/>
+			<span {...this.props}>
```

Comment:
Yes, I read that and thus select this method, they have an internal id, which is not html id at all